data/json/recipes/other/tool.json

```diff
@@ -216,6 +216,63 @@
     "qualities": [ { "id": "HAMMER", "level": 1 } ],
     "components": [ [ [ "steel_chunk", 1 ], [ "steel_lump", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "tinderbox",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "survival",
+    "skills_required": [ "fabrication", 2 ],
+    "difficulty": 2,
+    "time": "5 m",
+    "autolearn": true,
+    "book_learn": [ [ "mag_survival", 1 ], [ "textbook_survival", 1 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "can_drink_unsealed", 1 ], [ "can_food_unsealed", 1 ], [ "scrap", 2 ] ] ]
```

Comment:
Drop the scrap alternative, it needs a good bit more tooling than exists in this recipe to turn scrap into a functional box.